src/pages/constitution/Constitution_Full.tsx

```diff
@@ -79,10 +87,10 @@ class Constitution_Full extends React.Component<Props> {
     }
   }
 
-  shouldComponentUpdate(nextProps: Readonly<Props>, nextState: Readonly<{}>, nextContext: any): boolean {
-    // the view state never actually changes
-    return false;
-  }
+  // shouldComponentUpdate(nextProps: Readonly<Props>, nextState: Readonly<{}>, nextContext: any): boolean {
```

Comment:
This may the cause of the performance issue you're seeing. We can talk through it. The challenge is that React is pretty eager about re-rendering a page and, for very big pages like the full constitution, it does so repeatedly even though the state hasn't changed.